sdk/translation/azure-ai-translation-text/src/test/java/com/azure/ai/translation/text/TranslateTests.java

```diff
@@ -57,7 +57,7 @@ public void translateWithNoTranslateTag() {
         ArrayList<InputTextItem> content = new ArrayList<>();
         content.add(new InputTextItem("<span class=notranslate>今天是怎么回事是</span>非常可怕的"));
 
-        List<TranslatedTextItem> response = getTranslationClient().translate(targetLanguages, content, null, "zh-chs", TextType.HTML, null, null, null, null, null, null, null, null, null);
+        List<TranslatedTextItem> response = getTranslationClient().translate(targetLanguages, content, null, "zh-Hans", TextType.HTML, null, null, null, null, null, null, null, null, null);
```

Comment:
In GA, we may need to improve the API to options bag.

I assume `content` is the payload, most of others is the query param, traceId is header param.

Reply:
I will make a task for myself. Would you be able to find any existing API which is doing the same? Is it custom code in Java or does it require the TypeSpec change?

Reply:
At present, it had to be done in Java code.

If done by partial-update, one just remove the `@Generated` annotation over the method, and modify it in place (in this case, also need to add a e.g. `TranslateOptions` class with ctor/setter/getter.

If done by the customization class, it could be a bit more complex. But generally same idea of modifying the method and add a class.

There were a long discussion of whether we can do cross-language support for this case in typespec client.tsp. So far, no final decision yet.